recipe/meta.yaml

```diff
@@ -22,11 +22,12 @@ requirements:
   host:
     - pip
     - python
-    # bootstrapping
-    - packaging >=21.3
-    - pathspec >=0.10.1
-    - pluggy >=1.0.0
-    - tomli >=2.0
+    - wheel
+    - packaging 21.3
+    - pathspec 0.10.3
+    - pluggy 1.0.0
+    - tomli 1.2.2
+    - trove-classifiers 2023.3.9
```

Comment:
`editables` is missing from the list of requirements, see https://github.com/pypa/hatch/blob/master/backend/pyproject.toml#L37.